<launch>

<rosparam command="load" file="$(find fato_tracker_nodes)/config/tracker_parameters.yaml"/>
<param name="fato/model/h5_file" type="str" value="$(find fato_tracker_tests)/../data/ros_hydro/ros_hydro_features.h5"/>
<param name="fato/model/obj_file" type="str" value="$(find fato_tracker_tests)/../data/ros_hydro/ros_hydro.obj"/>

<!-- uncomment to use the medium version-->
 <node name="tracker" pkg="fato_tracker_tests" type="synthetic_track" output="screen">
 </node>
  
</launch>
